dataprofiler/profilers/histogram_utils.py

```diff
@@ -14,6 +14,293 @@
 )
 
 
+def _ptp(maximum, minimum):
```

Comment:
do we need all of the imports above? also, do we know why we `ignore`?

Reply:
We do use all of the imports above in the `get_bin_edges` function and no not sure why there is an ignore, probably because of reference to protected methods?

Reply:
we could update the ignore to specifically capture that condition as oppose to a totality